bindings/python/FloatingBaseEstimators.cpp

```diff
@@ -0,0 +1,147 @@
+        .def_readwrite("l_contact_frame_position", &StateStdDev::lContactFramePosition)
+        .def_readwrite("r_contact_frame_orientation", &StateStdDev::rContactFrameOrientation)
+        .def_readwrite("r_contact_frame_position", &StateStdDev::rContactFramePosition);
+
+    py::class_<Output>(module, "FloatingBaseEstimatorOutput")
+        .def(py::init())
+        .def_readwrite("state", &Output::state)
+        .def_readwrite("state_std_dev", &Output::stateStdDev)
+        .def_readwrite("base_pose", &Output::basePose)
+        .def_readwrite("base_twist", &Output::baseTwist);
+
+    py::class_<Measurements>(module, "FloatingBaseEstimatorMeasurements")
+        .def(py::init())
+        .def_readwrite("acc", &Measurements::acc)
+        .def_readwrite("gyro", &Measurements::gyro)
+        .def_readwrite("encoders", &Measurements::encoders)
+        .def_readwrite("encoder_speeds", &Measurements::encodersSpeed)
+        .def_readwrite("stamped_contact_status", &Measurements::stampedContactsStatus)
+        .def_readwrite("lf_in_contact", &Measurements::lfInContact)
+        .def_readwrite("rf_in_contact", &Measurements::rfInContact);
+
+    py::class_<Options>(module, "FloatingBaseEstimatorOptions")
+        .def(py::init())
+        .def_readwrite("imu_bias_estimation_enabled", &Options::imuBiasEstimationEnabled)
+        .def_readwrite("static_imu_bias_initialization_enabled", &Options::staticImuBiasInitializationEnabled)
+        .def_readwrite("nr_samples_for_imu_bias_initialization", &Options::nrSamplesForImuBiasInitialization)
+        .def_readwrite("ekf_update_enabled", &Options::ekfUpdateEnabled)
+        .def_readwrite("acceleration_due_to_gravity", &Options::accelerationDueToGravity);
+
+    py::class_<SensorsStdDev>(module, "FloatingBaseEstimatorSensorsStdDev")
+        .def(py::init())
+        .def_readwrite("accelerometer_noise", &SensorsStdDev::accelerometerNoise)
+        .def_readwrite("gyroscope_noise", &SensorsStdDev::gyroscopeNoise)
+        .def_readwrite("accelerometer_bias_noise", &SensorsStdDev::accelerometerBiasNoise)
+        .def_readwrite("gyroscope_bias_noise", &SensorsStdDev::gyroscopeBiasNoise)
+        .def_readwrite("contact_foot_lin_vel_noise", &SensorsStdDev::contactFootLinvelNoise)
+        .def_readwrite("contact_foot_ang_vel_noise", &SensorsStdDev::contactFootAngvelNoise)
+        .def_readwrite("swing_foot_lin_vel_noise", &SensorsStdDev::swingFootLinvelNoise)
+        .def_readwrite("swing_foot_ang_vel_noise", &SensorsStdDev::swingFootAngvelNoise)
+        .def_readwrite("forward_kinematics_noise", &SensorsStdDev::forwardKinematicsNoise)
+        .def_readwrite("encoders_noise", &SensorsStdDev::encodersNoise);
+
+    py::class_<Advanceable<Output>>(module, "FloatingBaseEstimatorOutputAdvanceable");
+
+    py::class_<FloatingBaseEstimator, Advanceable<Output>> floatingBaseEstimator(module, "FloatingBaseEstimator");
+
+    floatingBaseEstimator.def(py::init())
```

Comment:
Any reason to separate methods definition from the creation of the `py::class_` object?

Reply:
It was suggested as a way to handle nested classes or internal types.

I followed examples suggested in, 
- https://github.com/pybind/pybind11/issues/2032#issuecomment-707932778
- https://pybind11.readthedocs.io/en/stable/classes.html?highlight=const%20method#enumerations-and-internal-types

Reply:
Ok, only now I see that the object is used below, good to know!